### PR TITLE
Add build badges

### PR DESCRIPTION
**Pull request by flands**
- Reduce information in README in favor of docs links
- Add docs/monitoring.md
- Add docs/sizing.md
- Move security, troubleshooting and migrating info to docs

### CODE REVIEW

**Review by asuresh4 — APPROVED** [review]

File: docs/components.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+| k8s_cluster      | k8s_tagger        | sapm      | k8s_observer  |
+| kubeletstats     | memorylimiter     | signalfx  | pprof         |
+| opencensus       | metrictransform   | splunkhec | zpages        |
+| otlp             | resource          |           |               |
+| receiver_creator | resourcedetection |           |               |
+| sapm             | span              |           |               |
+| signalfx         |                   |           |               |
+| simpleprometheus |                   |           |               |
+| smartagent       |                   |           |               |
+| splunkhec        |                   |           |               |
```
Comment (asuresh4) [inline docs/components.md:25]: If we going by the config names of components in this table, `splunkhec` -> `splunk_hec`.

File: docs/components.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+removed prior to the 1.0 release.
+
+| Receivers        | Processors        | Exporters | Extensions    |
+| :--------------: | :--------:        | :-------: | :--------:    |
+| fluentforward    | attributes        | file      | healthcheck   |
+| hostmetrics      | batch             | logging   | httpforwarder |
+| jaeger           | filter            | otlp      | host_observer |
+| k8s_cluster      | k8s_tagger        | sapm      | k8s_observer  |
+| kubeletstats     | memorylimiter     | signalfx  | pprof         |
+| opencensus       | metrictransform   | splunkhec | zpages        |
```
Comment (flands) [inline docs/components.md:18]:
```suggestion
| opencensus       | metrictransform   | splunk_hec | zpages        |
```

File: docs/components.md (hunk 1 above)
Comment (flands) [inline docs/components.md:25]:
```suggestion
| splunk_hec       |                   |           |               |
```

Comment (flands) [reply docs/components.md]: Fixed!

**Review by asuresh4 — APPROVED** [review]

**Pull request merged**